### ISSUE TITLE
proper alignment to root

### ISSUE THREAD

**Issue by rneher** [blab/nextflu#120]
replace the current alignment algorithm with something that tolerates gaps

**Comment by trvrb** [comment]
Can you say more? You'd like an alignment that maintains gaps? Wouldn't numbering get very complicated in this case?

I guess you could do an alignment that allows gaps then build the tree then remove gaps to get canonical numbering.

**Comment by rneher** [comment]
this refers to the sequence search in javascript. We align to root using a stupid string matching routine and then look for mutations. If the query has a gap relative to root, no proper alignment is found.

**Comment by trvrb** [comment]
This has been resolved in the [nextstrain/augur](https://github.com/nextstrain/augur) codebase.

**Comment by rneher** [comment]
I don't think this is resolved. It also refers to sequence search/alignment in auspice rather than augur. This should be a feature request in auspice. I did write a proper pairwise aligner a while back: https://github.com/neherlab/timetree_viewer/blob/master/js/alignPairwise.js

**Comment by trvrb** [comment]
I see. Thanks Richard. Just reopened.
